modules/nw-bpfman-infw-about.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * networking/network_security/bpfman-operator/bpfman-operator-about.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ingress-node-firewall-operator_{context}"]
+= Ingress Node Firewall Operator integration
+
+The Ingress Node Firewall uses link:https://www.kernel.org/doc/html/latest/bpf/index.html[eBPF] programs to implement some of its key firewall functionality. By default these eBPF programs are loaded into the kernel using a mechanism specific to the Ingress Node Firewall. You can configure the Ingress Node Firewall Operator to use the bpfman Operator for loading and managing these programs instead.
+
+When this integration is enabled, the following limitations apply:
+
+- The Ingress Node Firewall Operator uses TCX if XDP is not available and TCX is incompatible with bpfman.
+- The Ingress Node Firewall Operator daemon set pods remain in the `CREATING` state until the firewall rules are applied.
```

Comment:
`ContainerCreating` I think that was the pod state name

Reply:
@msherif1234, fixed!